source/runner/Executor.ts

```diff
@@ -277,6 +277,7 @@ export class Executor {
         this.platform.deleteMainComment(dangerID)
       } else {
         const commitID = git.commits[git.commits.length - 1].sha
+        // TODO: GitLab template formatting (or reuse one of the others?)
```

Comment:
It's unique on danger ruby, so might need to be - https://github.com/danger/danger/tree/master/lib/danger/comment_generators

Reply:
I've checked on these but every comment I have seen so far looks fine. I can't get the `Generated by Danger` message to sit off to the right no matter the CSS styles.

<img width="832" alt="Screen Shot 2019-04-08 at 5 57 10 PM" src="https://user-images.githubusercontent.com/508403/55766078-c2cdc380-5a27-11e9-993a-af314ece6f96.png">
<img width="846" alt="Screen Shot 2019-04-08 at 5 57 07 PM" src="https://user-images.githubusercontent.com/508403/55766079-c2cdc380-5a27-11e9-8eb7-7a0838c00890.png">

Reply:
Maybe it doesn't do that on GitLab? (I always had not right aligned because it kinda felt like a signature) I'm not offended if it goes left aligned 👍